tests/mocks/limsmock.py

```diff
@@ -18,6 +21,17 @@ def __init__(self, label: str, sequence: str):
         self.sequence: str = sequence
 
 
+class LimsUDF(BaseModel):
+    control: str | None = None
+    customer: str = None
+
+    def get(self, argument: str) -> str:
+        if argument == "Control":
+            return self.control
+        if argument == "customer":
+            return self.customer
+
```

Comment:
What is the point of this get method?

Reply:
This is just a MockLimsUDF and the get function is a mock of the sample.udf.get() function used here: https://github.com/Clinical-Genomics/cg/blob/74e358cd86686be07c0cee876e3f73c94b924e64/cg/apps/lims/api.py#L537